app/services/scene-collections/nodes/sources.ts

```diff
@@ -240,4 +240,42 @@ export class SourcesNode extends Node<ISchema, {}> {
       Promise.all(promises).then(() => resolve());
     });
   }
+
+
+  migrate(version: number) {
+
+    // migrate audio sources names
+    if (version < 3) {
+
+      this.data.items.forEach(source => {
+
+        if (source.name === 'AuxAudioDevice1') {
+          source.name =  'Mic/Aux';
+          return;
+        }
+
+        if (source.name === 'DesktopAudioDevice1') {
+          source.name = 'Desktop Audio';
+          return;
+
+        }
```

Comment:
Are these needed with the more general Regex matchers below?

Reply:
Good catch. I just copied this code from the removed `dispayName` getter.